src/components/SingleTweet.tsx

```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
+  padding: 0 16px;
```

Comment:
margins and paddings in bootstrap can be managed in next way:
https://getbootstrap.com/docs/4.0/utilities/spacing/

please fix here and everywhere